### PR TITLE
HPCC-14734 UI items login/logout in ECL Watch

### PR DESCRIPTION
**Pull request by miguelvazq**
Create the landing page for session management integration.

Signed-off by: Miguel Vazquez <miguel.vazquez@lexisnexis.com>

### CODE REVIEW

**Comment by miguelvazq** [comment]
@GordonSmith Please review. FYI @wangkx

File: esp/src/eclwatch/templates/Login.html [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    		background-color: #13b1f9;
+    		color: white;
+		}
+	</style>
+</head>
+<body>
+	<div class="container">
+		<div class="formContainer">
+			<img src="../img/Loginlogo.png" alt="HPCC Systems">
+			<p>Please log into ECL Watch</p>
```
Comment (GordonSmith) [inline esp/src/eclwatch/templates/Login.html]: Needs translation string

File: esp/src/eclwatch/templates/Login.html [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+			<div class="login">
+				<form method="POST" action="/">
+					<input type="text" name="username" placeholder="username"></input>
```
Comment (GordonSmith) [inline esp/src/eclwatch/templates/Login.html]: Needs translation string

File: esp/src/eclwatch/templates/Login.html [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+					<input type="password" name="password" placeholder="password"></input>
```
Comment (GordonSmith) [inline esp/src/eclwatch/templates/Login.html]: Needs translation string

**Review by GordonSmith — CHANGES_REQUESTED** [review]
Hard to review it without seeing it on a web page - but it will need to handle the translation scheme.

**Comment by miguelvazq** [comment]
@GordonSmith Please review. Translations added. I am going to start adding Spanish translations as we go. I do however get this message about Conflicting files (see above).

**Comment by richardkchapman** [comment]
Needs rebasing for a clean merge

**Comment by miguelvazq** [comment]
@richardkchapman rebased.

**Comment by richardkchapman** [comment]
@GordonSmith This is waiting for you I think

**Review by GordonSmith — APPROVED** [review]
Looks fine.

**Pull request merged**